### DISCUSSION TITLE
Update for the satellite database doesnt work

### DISCUSSION THREAD

**Discussion by Jojo1703** [mworion/MountWizzard4#358] [Q&A] [answered]
Hello,

I tried an update for the satellite database, e.g. Space Stations, but it doesn't work. The process hangs and it is not even shown that the satellites would be updated, see picture.
![mwsat](https://github.com/mworion/MountWizzard4/assets/65235013/244bbb5f-551d-4d66-9274-9edf6cef1106)
[mw4-2024-06-25.log](https://github.com/user-attachments/files/15970245/mw4-2024-06-25.log)

greeting
Jurgen

**Comment by Jojo1703** [comment, marked as answer]
Hi,

I've found a solution for the satellite update issue. If you delete the satellite files, e.g., station.txt or visual.txt, in the ...workdir/data directory before starting MW4, the satellite data will be updated correctly the next time you start MW4.

Regards,

Jurgen

**Comment by mworion** [comment]
Hi Jürgen,

great to hear you solve it. I will add these hints to the Q&A part of the 
documentation.
Michel

Gesendet mit der Telekom Mail App
<http://www.t-online.de/service/redir/emailmobilapp_ios_smartphone_footerlink.htm>

-----Original-Nachricht-----
Von: Jojo1703 ***@***.*** ***@***.***> 
>
Betreff: Re: [mworion/MountWizzard4] Update for the satellite database 
doesnt work (Discussion #358)
Datum: 05.03.2026, 12:03 Uhr
An: mworion/MountWizzard4 ***@***.***
***@***.***> >
CC: Subscribed ***@***.***
***@***.***> >

Hi,

I've found a solution for the satellite update issue. If you delete the 
satellite files, e.g., station.txt or visual.txt, in the ...workdir/data 
directory before starting MW4, the satellite data will be updated correctly 
the next time you start MW4.

Regards,

Jurgen

—
Reply to this email directly, view it on GitHub
<https://github.com/mworion/MountWizzard4/discussions/358#discussioncomment-16013187> 
, or unsubscribe
<https://github.com/notifications/unsubscribe-auth/ABSTMFGJWHW3JFANBZJF7MT4PGXOZAVCNFSM6AAAAACWIP2J46VHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTMMBRGMYTQNY> 
.
You are receiving this because you are subscribed to this thread.Message 
ID: ***@***.***
com>
